server/src/main/kotlin/eu/kanade/tachiyomi/network/PersistentCookieStore.kt

```diff
@@ -32,48 +40,153 @@ class PersistentCookieStore(context: Context) {
 
     @Synchronized
```

Comment:
still needed with ReentrantLock? (same for `removeAll`)

Reply:
Done 👍